web/css/source/_theme.less

```diff
@@ -195,6 +226,7 @@ p,li,dt,dd,td,th,details,button,form,a,div,details {
     .content-container{
       padding-left: 15%;
       display: block;
+      celear:both;
```

Comment:
PHPStorm, I just plain and simple missed it - didn't help that I was doing most the adjustments in the block view code in Magento, so not looking at the IDE at all most of the time.